debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerMetrics.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.oracle.logminer;
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.AtomicReference;
+
+import io.debezium.annotation.ThreadSafe;
+import io.debezium.connector.common.CdcSourceTaskContext;
+import io.debezium.metrics.Metrics;
+
+/**
+ * This class contains methods to be exposed via MBean server
+ *
+ */
+@ThreadSafe
+public class LogMinerMetrics extends Metrics implements LogMinerMetricsMXBean {
+    private AtomicLong currentScn = new AtomicLong();
```

Comment:
Can be final? Also metrics need discussion in docs?

Reply:
I've marked these as final.  I'll update the DBZ-137 docs PR and add information about metrics there.